src/main/java/edu/greenblitz/robotName/subsystems/Arm/ElbowUtils/SimulationElbow/SimulationElbow.java

```diff
@@ -0,0 +1,82 @@
+package edu.greenblitz.robotName.subsystems.Arm.ElbowUtils.SimulationElbow;
+
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import edu.greenblitz.robotName.RobotConstants;
+import edu.greenblitz.robotName.subsystems.Arm.ElbowUtils.ElbowConstants;
+import edu.greenblitz.robotName.subsystems.Arm.ElbowUtils.ElbowInputsAutoLogged;
+import edu.greenblitz.robotName.subsystems.Arm.ElbowUtils.IElbow;
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.wpilibj.simulation.SingleJointedArmSim;
+import org.littletonrobotics.junction.Logger;
+
+import static edu.greenblitz.robotName.RobotConstants.SimulationConstants.*;
+
+public class SimulationElbow implements IElbow {
+
+    SingleJointedArmSim elbowSimulation;
```

Comment:
Why not private?

![image](https://github.com/GreenBlitz/CRRRRESCENDO/assets/80216549/6bf32099-a878-4137-ad95-0347ce55535d)